packetbeat/docs/gettingstarted.asciidoc

```diff
@@ -78,7 +78,7 @@ PS C:\Program Files\Packetbeat> .\install-service-packetbeat.ps1
 NOTE: If script execution is disabled on your system, you need to set the execution policy for the current session to allow the script to run. For example: `PowerShell.exe -ExecutionPolicy UnRestricted -File .\install-service-packetbeat.ps1`. 
 
 Before starting Packetbeat, you should look at the configuration options in the
-configuration file, for example `C:\Program Files\Packetbeat\packetbeat.yml`. For
+configuration file, for example `C:\Program Files\Packetbeat\packetbeat.yml` or `/etc/Packetbeat/packetbeat.yml`. For
```

Comment:
Yes, it's lowercase. (It's uppercase on Windows only because the installation instructions recommend changing the name of the folder.)